Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.13-slim
+
+WORKDIR /app
+
+COPY requirements.txt .
```

Comment:
Ensure that `requirements.txt` exists in the build context. If it doesn't, the `COPY` command will fail.